### ISSUE TITLE
Provide setting so that asterisk always looks the same

### ISSUE THREAD

**Issue by bradyt** [vector-im/element-web#21376]
### Your use case

The asterisk here is displayed at varying heights. I would like an option in settings to prevent that behavior. I just want to see the same asterisk all the time.

<img width="111" alt="asterisk-hi-lo" src="https://user-images.githubusercontent.com/6107051/157666290-7eb89148-d661-4e45-a494-f1e00ac5992e.png">

### Have you considered any alternatives?

I switched the font to Menlo, and it seemed to fix it.

**Comment by bradyt** [comment]
Similarly for "x".

<img width="102" alt="x-hi-lo" src="https://user-images.githubusercontent.com/6107051/157667305-fe86791c-7336-49ed-a690-3d05c5a2f3c0.png">

**Comment by t3chguy** [comment]
This is a feature of the font. Pretty sure browsers don't allow us to modify the font to tweak such features.
You can change fonts in Settings > Appearance.

Related https://github.com/rsms/inter/issues/404

**Comment by bradyt** [comment]
I asked about this in chat, thank you for discussion there. I had not realized this was a feature of a font. I was imagining that Element was dynamically selecting characters for display or replacement.

My confusion was compounded by my attempts to avoid modification of text interpreted as Markdown, like in "3*4*5". To me, it looked like Element was continuing this behavior when I added spaces. I have never deliberately seeked out fonts with these features, so I didn't have experience and knowledge to interpret what I was seeing. I was then spending time trying to check what the text looked like to the receiver, as well as copying and pasting the text to other apps to compare the two versions of text.

Given the context, of IRC bridges, lack of an option to disable Markdown, and the fact that some of us don't know the subtleties of fonts displaying ASCII characters as completely different non-ASCII characters, or expecting such to be set as default. I protest the choice of Element setting the default font with such features.

I am not insisting on changing the default, but wanted to leave a more complete note with the closed issue. The chat was helpful in expanding this note.

For those looking for the quick fix: above, I used Menlo just as my initial test. Using monospace font didn't seem like a good implicit assumption about how the app was designed, so I found what seems like a more canonical solution, of putting "font: system-ui".
